WEBAplication/Controllers/FlightController.cs

```diff
@@ -0,0 +1,92 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using WebApplication1.Models;
+using WebApplication1.Storage;
+
+namespace WebApplication1.Controllers
+{
+    [Route("api")]
+    [ApiController]
+    public class FlightController : ControllerBase
+    {
+        [Route("airports")]
+        [HttpGet]
+
```

Comment:
šis visur jāsalabo

Reply:
Kas tieši būtu jāmaina no  [Route("airports")]?

Reply:
```suggestion
```

Reply:
es komentāru ieliku precīzi tajā rindā kurā bija problēma. :D